### PR TITLE
feat: Documentation update

### PR DESCRIPTION
**Pull request by vitalyrudenko-affinidi**
https://github.com/affinidi/reference-app-portable-reputation/blob/documentation-update/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,185 @@
...
+2. Install the dependencies:
+    ```
+    $ npm install
+    ```
+3. Create a `.env` file:
+    ```
+    $ cp .env.example .env
+    ```
+    Please enter values for `PROJECT_ID`, `PROJECT_DID` and `API_KEY_HASH` from your project.  
+    You also need to [create a Github OAuth app](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app) and enter values for `GITHUB_APP_CLIENT_ID` and `GITHUB_APP_CLIENT_SECRET`.
```
Comment (carlos-affinidi) [inline README.md:30]: Let's add emphasis here

File: README.md [hunk 2]
```diff
@@ -1,36 +1,185 @@
...
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+## Tools & frameworks
+
+This project is built with **NextJS** framework, which allows you to quickly build apps using TypeScript and **React**. NextJS has built-in router, server-side rendering, backend support and useful extensions like Next Auth for authenticating via Github and other data providers.  
+Read [NextJS docs](https://nextjs.org/docs/getting-started), [React docs](https://reactjs.org/docs/getting-started.html).  
+
+We also use **Styled Components**, **Tailwind CSS** and Bootstrap Grid to build the UI.  
+Read [Styled Components docs](https://styled-components.com/docs), [TailWind CSS docs](https://tailwindcss.com/docs/installation).
```
Comment (carlos-affinidi) [inline README.md:124]: We don't use Tailwind in Portable Reputation app yet

File: README.md (hunk 1 above)
Comment (vitalyrudenko-affinidi) [reply README.md:30]: What do you have in mind?

Comment (carlos-affinidi) [reply README.md]: Just to make this more visible. Bold or italics or both. A line break as well.

Comment (vitalyrudenko-affinidi) [reply README.md]: Added, please check

File: README.md [hunk 3]
```diff
@@ -1,36 +1,190 @@
...
+  App->>-User: Show "Github profile" page with GithubProfile VC's contents
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+## Tools & frameworks
+
+This project is built with **NextJS** framework, which allows you to quickly build apps using **TypeScript** and **React**. NextJS has built-in router, server-side rendering, backend support and useful extensions like Next Auth for authenticating via Github and other data providers.  
+Read [NextJS docs](https://nextjs.org/docs/getting-started), [React docs](https://reactjs.org/docs/getting-started.html).  
+
+We also use **Styled Components** and **Bootstrap Grid** to build the UI.
```
Comment (carlos-affinidi) [inline README.md:128]:
```suggestion
We also use **Styled Components** and **Tailwind CSS** to build the UI. 
```

File: README.md [hunk 4]
```diff
@@ -1,36 +1,190 @@
...
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+## Tools & frameworks
+
+This project is built with **NextJS** framework, which allows you to quickly build apps using **TypeScript** and **React**. NextJS has built-in router, server-side rendering, backend support and useful extensions like Next Auth for authenticating via Github and other data providers.  
+Read [NextJS docs](https://nextjs.org/docs/getting-started), [React docs](https://reactjs.org/docs/getting-started.html).  
+
+We also use **Styled Components** and **Bootstrap Grid** to build the UI.  
+Read [Styled Components docs](https://styled-components.com/docs).
```
Comment (carlos-affinidi) [inline README.md:129]:
```suggestion
Read [Styled Components docs](https://styled-components.com/docs), [Tailwind CSS docs](https://tailwindcss.com/docs/installation).
```

**Pull request merged**